src/main/java/com/redhat/devtools/intellij/common/utils/ExecHelper.java

```diff
@@ -431,7 +433,7 @@ public static void linkProcessToTerminal(Process p, Project project, String titl
       try {
         ensureTerminalWindowsIsOpened(project);
         boolean isPost2018_3 = ApplicationInfo.getInstance().getBuild().getBaselineVersion() >= 183;
```

Comment:
This can be removed as it is not used anymore.

Reply:
I'll fix this and also correct the `NotificationGroupAdapter` and re-submit